### PR TITLE
do not quantize logits matmul

### PR DESCRIPTION
**Pull request by ychzhang**

### CODE REVIEW

File: MaxText/configs/base.yml [hunk 1]
```diff
@@ -40,10 +40,11 @@ gcs_metrics: False
 # Activation dtypes.
 dtype: "bfloat16"
 int8_training: False
-# The following three flags will only have effect when int8_training is True
+# The following four flags will only have effect when int8_training is True
 use_dqdg: False
 fwd_int8: True
 bwd_int8: True
+quantize_logits: False
```
Comment (lukaszlew) [inline MaxText/configs/base.yml:47]: change to true.
